libs/block-mesh-manager/migrations/20240822174812_create_analytics_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE analytics
+(
+    user_id          UUID        NOT NULL REFERENCES users (id),
+    depin_aggregator TEXT        NOT NULL,
```

Comment:
i think this can be null, incase we want to use it for other purposes in the future

Reply:
ok keeping not null, we'll just use empty string if not relevant.